instrumentation/jetty/jetty-8.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jetty/v8_0/Jetty8HandlerAdvice.java

```diff
@@ -46,6 +55,17 @@ public static void stopSpan(
       @Advice.Local("otelContext") Context context,
       @Advice.Local("otelScope") Scope scope) {
 
-    JettyHandlerAdviceHelper.stopSpan(tracer(), request, response, throwable, context, scope);
+    if (scope == null) {
+      return;
+    }
+    scope.close();
+
+    if (throwable != null) {
+      if (response.getStatus() < 400) {
+        response.setStatus(500);
```

Comment:
This feels a bit scary. It would be best if we didn't need to change response status, kind of hard to tell what behavioural changes it might cause.